kibana.yml

```diff
@@ -32,6 +32,8 @@
 # is proxied through the Kibana server.
 # elasticsearch.username: "user"
 # elasticsearch.password: "pass"
+%ELASTICSEARCH_USERNAME%
```

Comment:
Should probably use a different name then, given the user specified env var is also called `ELASTICSEARCH_USERNAME`?